lib/common.sh

```
@@ -1,4 +1,8 @@
-# Common utility functions for shiplog-lite
+# Common utility functions for Shiplog CLI
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)